web/electron/application-menu.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ *  Copyright (c) 2021 the Octant contributors. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { app, Menu, shell } from 'electron';
+import { errLogPath } from './paths';
+
+export class ApplicationMenu {
```

Comment:
This needs to be unit tested, especially now when it's a separate class. I'd be very nervous to release it without automated tests.

Reply:
I'm punting on this. There's no headless support for electron unless we compile our own. Otherwise we will add spectron/mocha as a requirement for 017